assets/js/components/settings/SettingsActiveModule/ConfirmDisconnect.js

```diff
@@ -73,7 +73,7 @@ export default function ConfirmDisconnect( { slug, handleDialog } ) {
 			clearWebStorage();
 			global.location.assign( adminReauthURL );
 		}
-	}, [ module?.slug ] );
+	}, [ adminReauthURL, deactivateModule, module.forceActive, slug ] );
```

Comment:
as this is a `useCallback` and not `useEffect` then IMHO we can just follow `eslint-plugin-react-hooks` recommendations (it's not re-running code on any change here, it's just assuring there are no stale closures)

Reply:
Fixed @tofumatt